js_assets/Question.vue

```diff
@@ -0,0 +1,150 @@
+<template>
```

Comment:
I think what I had in mind for the help steps was more like a button for the hint -> show the hint with a button to see the answer -> show the answer with either a button to see the next hint or to get the next question.  I would like the help steps to remain on the screen so they can see all of the help steps/answers while they are still on the question.  In other words, I don't want the previous help steps to go away when a new help step shows.   I also want to make it more deliberate to get the answer so they don't accidentally hover over the answer before they've had a chance to think about it.